### DISCUSSION TITLE
Firebase push notification with flutter

### DISCUSSION THREAD

**Discussion by nuaiman** [pocketbase/pocketbase#2494] [Q&A]
Hello I want to implement firebase push notification with pocketbase for my app, this is because I heard Firebase push notification is fully free, could someone teach me on how to implement it for a flutter app. 

PS - if any other service is fully free I dont mind using it as long as it supports android, ios at the least.

Thanks

**Comment by ganigeorgiev** [comment]
I don't see how this is related to PocketBase. You can refer to the official Firebase docs for examples and guides how to integrate with Flutter - https://firebase.google.com/docs/cloud-messaging/flutter/client.
